strider/fetcher.py

```diff
@@ -124,10 +124,17 @@ async def setup(
             self.preferred_prefixes,
         ))["query_graph"]
 
-        # Expand the query graph using the biolink model hierarchies
+        # Fill in missing categories and predicates using normalizer
         await fill_categories_predicates(self.qgraph, self.logger)
         standardize_graph_lists(self.qgraph)
 
+        # Replace biolink:Protein with biolink:GeneOrGeneProduct
+        for node in self.qgraph["nodes"].values():
+            categories = node.get("category", [])
```

Comment:
I think this can be done more cleanly. Something like this:
```python
if not node.get("category"):
    continue
node["category"] = [
    "biolink:GeneOrGeneProduct" if category == "biolink:Protein" else category
    for category in node["category"]
]
```